frontend/src/pages/offers/PalletsEditorSidebar.tsx

```diff
@@ -0,0 +1,121 @@
+import cx from 'classnames'
+import { FunctionComponent } from 'react'
+import Button from '../../components/Button'
+import ChevronIcon from '../../components/icons/ChevronIcon'
+import WarningIcon from '../../components/icons/WarningIcon'
+import { OfferQuery, PalletQuery } from '../../types/api-types'
+import { validatePalletContents } from '../../utils/data'
+
+type Props = {
+  pallets: OfferQuery['offer']['pallets'] | undefined
+  selectedPalletId?: number
+  selectedLineItemId?: number
+  isLoading?: boolean
+  selectedPalletData?: PalletQuery
+  createPallet: () => void
+  selectPallet: (palletId: number) => void
+  addLineItem: (palletId: number) => void
+  selectLineItemId: (lineItemId: number) => void
+}
+
+const PalletsEditorSidebar: FunctionComponent<Props> = ({
```

Comment:
I moved the entire sidebar from the PalletsEditor file to this one. No functional changes.